en/_posts/2012-05-05-video-of-april-workshop.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Video: Rails Girls Workshop Berlin April 2012"
+author: "Ferdous"
+categories: en blog
+date: "2012-05-05"
+---
+
+Have you seen our (Rails Girls Berlin) April 2012 Workshop video? Alexander Lang made an awesome video from our Workshop. Check it out, maybe you’ll even find yourself in it!
+
+Enjoy ♥️
+
+<iframe src="http://player.vimeo.com/video/40852182?title=0&amp;byline=0&amp;portrait=0" frameborder="0" width="500" height="281"></iframe>
```

Comment:
This will break HTTPS of codecurious.org, please switch to HTTPS for vimeo. I confirmed the link still works then